templates/pages/product.html

```diff
@@ -27,7 +27,7 @@
             {{> components/products/videos product.videos}}
         {{/if}}
 
-        {{#all settings.show_product_reviews (if theme_settings.show_product_reviews_tabs '!==' true)}}
+        {{#all settings.show_product_reviews theme_settings.show_product_reviews (if theme_settings.show_product_details_tabs '!==' true)}}
```

Comment:
Can we also remove the `'!==' true` here for the `show_product_details_tabs` ?

Reply:
Are there any more elegant way to check if the flag is falsy? If not, it will break a condition logic